packages/core/workers/src/WorkerFarm.js

```diff
@@ -302,6 +316,12 @@ export default class WorkerFarm extends EventEmitter {
     );
   }
 
+  createReverseHandle(fn: () => mixed) {
+    let handle = new Handle();
+    this.handles.set(handle.id, fn);
+    return handle;
```

Comment:
can we return a disposable here in addition to the handle to run `this.handles.delete(handle.id)` when Parcel no longer needs it?

Reply:
Discussed. Will cover this in a separate PR.